ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp

```diff
@@ -99,6 +166,8 @@ void AirsimROSWrapper::initialize_ros()
 
     statistics_timer_ = nh_private_.createTimer(ros::Duration(1), &AirsimROSWrapper::statistics_timer_cb, this);
     go_signal_timer_ = nh_private_.createTimer(ros::Duration(1), &AirsimROSWrapper::go_signal_timer_cb, this);
+
+    publish_track();

```

Comment:
Sooo the track is published once? Is this desired behavior? Can we document this?